### PR TITLE
chore: add husky

### PR DESCRIPTION
**Pull request by pyphilia**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@ yarn-error.log*
 
 # development environments
 .idea/
+
+# react-app-env
+src/react-app-env.d.ts
```
Comment (juancarlosfarah) [inline .gitignore:26]: This is new, right?

**Review by juancarlosfarah — APPROVED** [review]
Looks good to me! Just one question: what is the `react-app-env.d.ts` file?

**Comment by pyphilia** [comment]
@juancarlosfarah From this issue https://github.com/facebook/create-react-app/issues/6560 It is an automatically generated file because of typescript. It seems necessary yet it contains nothing. Many included it in gitignore, and since it generates automatically it should not be a problem. But we could also include it in the project, let me know what you think.

**Comment by juancarlosfarah** [comment]
@pyphilia, no problem. Sounds like it should be ignored!

**Pull request merged**